hack/print-workspace-status.sh

```diff
@@ -40,4 +40,9 @@ STABLE_gitTreeState ${KUBE_GIT_TREE_STATE-}
 STABLE_gitVersion ${KUBE_GIT_VERSION-}
 STABLE_gitMajor ${KUBE_GIT_MAJOR-}
 STABLE_gitMinor ${KUBE_GIT_MINOR-}
+gitCommit ${KUBE_GIT_COMMIT-}
```

Comment:
to answer my own question: having the `STABLE_` lines seems to be enough to trigger relinking.

we should still probably get rid of the `STABLE_gitCommit` lines etc.

also, I note that the BuildDate isn't being set - is there any way we can easily fill in that tag too? `BUILD_TIMESTAMP` is set to epoch milliseconds in volatile-status.txt.